### DISCUSSION TITLE
Questions about using EP 22621.2506.60.1 after updating from Windows 11 21H2 to 22H2.

### DISCUSSION THREAD

**Discussion by CladInShadows99** [valinet/ExplorerPatcher#2563] [Q&A] [answered]
Hello! I'm on Windows 11 21H2 Build 22000.2538 and I am using ExplorerPatcher Version 22621/.2506.60.1 and I was wondering if it was safe to update my 21H2 Windows 11 version to 22H2 with ExplorerPatcher still installed. Should I uninstall it first? Is there any version of 22H2 that it will not work on? I appreciate any and all advice and help, thanks!

**Comment by pyrates999** [comment, marked as answer]
Uninstall EP first.

**Reply by CladInShadows99** [reply]
Thanks for the reply! I want to uninstall EP then update 2 22H2 but I was wondering, is there a way to backup my EP settings before uninstalling it so that I can restore them on a new install?

**Reply by pyrates999** [reply]
In the EP properties application, go to about.  Look for export, import, and restore to default.

**Comment by CladInShadows99** [comment]
Thank you! I uninstalled before updating to 22H2 then reinstalled and imported my settings. All seems fine. I appreciate the help!
